### ISSUE TITLE
The output path must be specified in the arguments for Swift and Scala

### ISSUE THREAD

**Issue by jordanhbuiltbyhq** [apollographql/apollo-ios#761]
I'm receiving the following error with version 0.15.2:

```
Error: The output path must be specified in the arguments for Swift and Scala
Command PhaseScriptExecution failed with a nonzero exit code
```

```
SCRIPT_PATH="${PODS_ROOT}/Apollo/scripts"
cd "${SRCROOT}/${TARGET_NAME}"
"${SCRIPT_PATH}"/check-and-run-apollo-cli.sh codegen:generate --target=swift --includes=./**/*.graphql --localSchemaFile="schema.json" API.swift
```

I did delete the local `node_modules` directory. `apollo -v` shows `apollo/2.18.2 darwin-x64 node-v8.15.0`. Installed via CocoaPods and using Xcode 11 GM. Let me know what more info you need!

**Comment by abernix** [comment]
My fly-by hunch: https://github.com/apollographql/apollo-tooling/commit/7e273f7539b9d8749873299587bf2f0803704fac#r35069355

**Comment by designatednerd** [comment]
Update: It turns out it wasn't that, it was an issue where when `--localSchemaFile` was the last argument before the output path, an updated tool to allow ingestion of multiple local schema files was seeing `API.swift` as a second schema file being passed in (that's also where we were getting that weird warning about the output path). Tooling team is working on a fix/workaround.

**Comment by designatednerd** [comment]
Updated CLI (and CLI installation instructions) shipped with `0.16.0`
